### PR TITLE
Ready

### PR DESCRIPTION
**Pull request by SergiyGovorukhin**

### CODE REVIEW

File: src/com/geekhub/hw5/translator/Translator.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        return "https://translate.yandex.net/api/v1.5/tr/translate?key=" + YANDEX_API_KEY + "&text=" + encodeText(text) + "&lang=" + TRANSLATION_DIRECTION;
+    }
+
+    /**
+     * Parses content returned by Yandex Translate API service. Removes all tags and system texts. Keeps only translated text.
+     * @param content that was received from Yandex Translate API by invoking prepared URL
+     * @return translated text
+     */
+    private String parseContent(String content) {
+        String regexp = "<text>(.+)?</text>";
```
Comment (SergiyGovorukhin) [inline src/com/geekhub/hw5/translator/Translator.java:56]: Some problems with this regexp.. In some cases it doesnt recognize < / text > part of file. May be it is better to use some XML parser java classes?

File: src/com/geekhub/hw5/translator/source/FileSourceProvider.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    @Override
+    public String load(String pathToSource) throws IOException {
+        String text = "";
+        File file = new File(pathToSource);
+        try(BufferedReader reader = new BufferedReader(new FileReader(file), BUFFER_SIZE)) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                text = text.concat(line).concat(" ");
```
Comment (SergiyGovorukhin) [inline src/com/geekhub/hw5/translator/source/FileSourceProvider.java:30]: I would better to use StringBuilder or StringBuffer, remembered after PR is done

Comment (AndrewBereza) [inline src/com/geekhub/hw5/translator/source/FileSourceProvider.java:30]: it's better to use Files NIO classes and methods, like Path and Files.readAllLines

**Comment by AndrewBereza** [comment]
Good job!

**Pull request closed without merging**